lib/puma/runner.rb

```diff
@@ -69,7 +69,7 @@ def start_control
 
       control.binder.parse [str], self, 'Starting control server'
 
-      control.run thread_name: 'control'
+      control.run thread_name: 'ctl'
```

Comment:
Suggestion: lets not change any thread names that are always less than 15 characters.

Reply:
The problem is that this thread name is also used as a prefix for its accompanying threadpool threads. So if we don't shorten it, we end up with names like `puma control tp 001` which is longer than 15 characters.